lesson-15/select-age.js

```diff
@@ -0,0 +1,105 @@
+export function initDropdownList() {
+    const select = document.createElement("select");
+    select.classList.add("child-age-select");
+    select.innerHTML = `<select>
+                        <option>0 years old</option>
+                        <option>1 year old</option>
+                        <option>2 years old</option>
+                        <option>3 years old</option>
+                        <option>4 years old</option>
+                        <option>5 years old</option>
+                        <option>6 years old</option>
+                        <option>7 years old</option>
+                        <option>8 years old</option>
+                        <option>9 years old</option>
+                        <option>10 years old</option>
+                        <option>11 years old</option>
+                        <option>12 years old</option>
+                        <option>13 years old</option>
+                        <option>14 years old</option>
+                        <option>15 years old</option>
+                        <option>16 years old</option>
+                        <option>17 years old</option>                   
+                        </select>`;
+      selectsWrapper.appendChild(select);
+  }
+  btnCounterChildrenPlus.addEventListener("click", initDropdownList);
+  
+  btnCounterChildrenPlus.addEventListener("click", () => {
+    switch (counterChildren.value) {
+      case "2":
+        childAgeQuastion.textContent = "What is the age of the children you’re travelling with?";
+        break;
+      case "3":
```

Comment:
the same. You don't have this variables in this file. Try make your main.js as shorter as possible. Take out all the variables, functions from it, and there just import and paste